school_teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './school_person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] Constructor extends parent's constructor by adding @specialization and a parameter for it. like this:
````ruby
 def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
````
 Please implement it

Reply:
Thank you for your corrections, I didn't understand what they meant by "extends" in that requirement but now I do.